.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit $1
```

Comment:
In the old setting the commit message was checked with the "-E" option.
Is this not necessary anymore?

Reply:
--edit replaces that